### PR TITLE
[GNA] Fix compilation

### PR DESCRIPTION
**Pull request by dorloff**
### Details:
 - Fix compilation error for GNA plugin

### Tickets:
 - 104500

### CODE REVIEW

**Review by evkotov — APPROVED** [review]

File: src/plugins/intel_gna/src/frontend/model_quantizer.hpp [hunk 1]
```diff
@@ -148,7 +148,7 @@ class ModelQuantizer {
                 inf_loop_count++;
             } else {
                 if (inf_loop_count > 0 &&
```
Comment (andrei-kochin) [inline src/plugins/intel_gna/src/frontend/model_quantizer.hpp:150]:
```suggestion
                if ((inf_loop_count > 0) &&
```

**Comment by dorloff** [comment]
#16719 is created instead to revert the problematic PR (#16003)

File: src/plugins/intel_gna/src/frontend/model_quantizer.hpp (hunk 1 above)
Comment (dorloff) [reply src/plugins/intel_gna/src/frontend/model_quantizer.hpp]: Thanks @andrei-kochin . We decided that we need to check this more carefully and revert first.

**Comment by rkazants** [comment]
@dorloff. still not

**Pull request closed without merging**